carbon-design-system/carbon-platform
# [web-app] page load improvements

- Improve page load times by using dynamic imports for large chunks (pictograms and icons metadata)
- Fix some typos
- Remove library provider and context in favor of a prop
- Remove IDL-site conditional from icons action bar

Fixes #1201

#### Testing / reviewing

- Ensure feature parity between old icons and pictograms pages and new ones
- Run a production build of the web-app `npm -w services/web-app run build`, run the production build locally `npm -w services/web-app run start`, and ensure all pages load faster than before (depending on network speed)
